roles/neutron-common/defaults/main.yml

```diff
@@ -56,6 +56,16 @@ neutron:
     # enabled can be "smart" or true / false. Smart will turn on lbaas if
     # controllers are dedicated.
     enabled: smart
+    haproxy:
+      lb_name: loadbalancer_name
+      usergroup: usergroup
+      sock_path: sock_path
```

Comment:
Why are we defining these variables in the roles and envs?
Why would customer need to change this?